server.sh

```diff
@@ -0,0 +1 @@
+python3 -m http.server
```

Comment:
This is my only thing to comment on 😆 ...

Is this file needed? @seanyoung247

Reply:
Oh, no. I added it because it's a pain to keep typing python3 -m http.server when testing locally. I'll remove.